css/properties.json

```diff
@@ -4733,6 +4749,54 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/field-sizing"
   },
+  "fill": {
+    "syntax": "none | <color> | <url> [none | <color>]? | context-fill | context-stroke",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "byComputedValueType",
+    "percentages": "no",
+    "groups": [
+      "Scalable Vector Graphics"
+    ],
+    "initial": "black",
+    "appliesto": "limitedSVGElementsShapeText",
+    "computed": "asColorOrAbsoluteURL",
+    "order": "perGrammar",
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/fill"
+  },
+  "fill-opacity": {
+    "syntax": "<alpha-value>",
+    "media": "visual",
+    "inherited": false,
```

Comment:
I think this inherits: https://drafts.fxtf.org/fill-stroke-3/#fill-opacity or am I looking at the wrong place?

Reply:
good catch.
but https://svgwg.org/svg2-draft/painting.html#StrokeOpacity is the better spec.

Reply:
Right, gotcha, this is confusing with the SVG/CSS crossover